src/swaps/blindex/BlindexConfig.js

```diff
@@ -0,0 +1,93 @@
+      token1: '0x99ac494Badd0CBa26143bd423E39A088591C7B09',
+      token0Symbol: 'ETHs',
+      token1Symbol: 'BDEU'
+    },
+    {
+      address: '0xdc7F0077E1921dBBCf8D042EE49184716301EDFE',
+      token0: '0x99ac494Badd0CBa26143bd423E39A088591C7B09',
+      token1: '0xB450ff06d950eFA9A9c0aD63790C51971C1BE885',
+      token0Symbol: 'BDEU',
+      token1Symbol: 'BDUS'
+    },
+    {
+      address: '0x39Eb6A2601aC29526E37EA60a531223AB0679873',
+      token0: '0x6542a10E68cEAc1Fa0641ec0D799a7492795AAC1',
+      token1: '0xB450ff06d950eFA9A9c0aD63790C51971C1BE885',
+      token0Symbol: 'BDX',
+      token1Symbol: 'BDUS'
+    },
+    {
+      address: '0x6a11d6af629bBbf310B82312E5fbd2dcc997a781',
+      token0: '0x542fDA317318eBF1d3DEAf76E0b632741A7e677d',
+      token1: '0xB450ff06d950eFA9A9c0aD63790C51971C1BE885',
+      token0Symbol: 'WRBTC',
+      token1Symbol: 'BDUS'
+    },
+    {
+      address: '0x4B481D8024B55263455AD3329e8be62C03A3F315',
+      token0: '0x1D931Bf8656d795E50eF6D639562C5bD8Ac2B78f',
+      token1: '0xB450ff06d950eFA9A9c0aD63790C51971C1BE885',
+      token0Symbol: 'ETHs',
+      token1Symbol: 'BDUS'
+    },
+    {
+      address: '0x2De59550090353D89fE36c50482234D9B7DB2e3B',
+      token0: '0xB450ff06d950eFA9A9c0aD63790C51971C1BE885',
+      token1: '0xb5999795BE0EbB5bAb23144AA5FD6A02D080299F',
+      token0Symbol: 'BDUS',
+      token1Symbol: 'XUSD'
+    }
+  ],
+  ERC20_INFO: [
+    { address: '0x6542a10E68cEAc1Fa0641ec0D799a7492795AAC1', symbol: 'BDX', decimals: 18 },
+    { address: '0x542fDA317318eBF1d3DEAf76E0b632741A7e677d', symbol: 'WRBTC', decimals: 18 },
+    { address: '0x1D931Bf8656d795E50eF6D639562C5bD8Ac2B78f', symbol: 'ETHs', decimals: 18 },
+    { address: '0xb5999795BE0EbB5bAb23144AA5FD6A02D080299F', symbol: 'XUSD', decimals: 18 },
+    { address: '0x99ac494Badd0CBa26143bd423E39A088591C7B09', symbol: 'BDEU', decimals: 18 },
+    { address: '0xB450ff06d950eFA9A9c0aD63790C51971C1BE885', symbol: 'BDUS', decimals: 18 },
+    { address: '0x15f2F01159a73A56a7149096942Ae4e2c019cbEf', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0x3e64c5DC35546D6feF70346A395ed8E21C4A5247', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0x2a5DC95E2F3150Ed52D69aD885EF6844dBED75C8', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0x00ffA27B58312A06dbc4aEd883d5F2EA924b2F89', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0xAB53CC054Ac0722aB0cBF346495E64c8C8e0c346', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0xdc7F0077E1921dBBCf8D042EE49184716301EDFE', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0x39Eb6A2601aC29526E37EA60a531223AB0679873', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0x6a11d6af629bBbf310B82312E5fbd2dcc997a781', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0x4B481D8024B55263455AD3329e8be62C03A3F315', symbol: 'UNI-V2', decimals: 18 },
+    { address: '0x2De59550090353D89fE36c50482234D9B7DB2e3B', symbol: 'UNI-V2', decimals: 18 }
```

Comment:
Why do we need all the LP tokens as well?

Reply:
Will change it to just wrbtc address and list of possible pair symbols.

Reply:
Removed LP tokens and decimals information